util/stream.go

```diff
@@ -64,9 +67,8 @@ func NewMessageStream(conn net.Conn, parser Parser) *MessageStream {
 	go m.outbound()
 	go m.inbound()
 
-	for i := 0; i < 25; i++ {
```

Comment:
@jojimt: There's at least an issue with that parallelism. The first issue is that there can only be 50 unread messages buffered in the channel. This means there's no point in having 25 worker goroutines to execute this parsing. Something like this would make sense in the context of a slow code path which takes at least a few milliseconds to run. This code would be blocked on the channel send as soon as 50 messages are in the queue if the messages are produced faster than they're consumed. On the consumer side in ofnet, we have only one reader process/goroutine, not 25.

The ofnet tests pass. I've also tested this change with netplugin unit tests, integration tests and system tests. The tests haven't taken longer to run because of this change either. I can imagine this may have been a potential optimization to address some channel related latency issue in Go.